src/models/calendar-restful-model-collection.ts

```diff
@@ -105,4 +105,68 @@ export default class CalendarRestfulModelCollection extends RestfulModelCollecti
         return Promise.reject(err);
       });
   }
+
+  consecutiveAvailability(
+    options: {
+      emails: Array<{
+        email: string;
+        name: string;
+      }>;
+      duration: number;
+      interval: number;
+      start_time?: string;
+      startTime?: string;
+      end_time?: string;
+      endTime?: string;
+      free_busy?: Array<{
+        emails: string;
+        object: string;
+        time_slots: Array<{
+          object: string;
+          status: string;
+          start_time: string;
+          end_time: string;
+        }>;
+      }>;
+      open_hours: Array<{
+        emails: Array<{
+          email: string;
+          name: string;
+        }>;
+        days: string[];
+        timezone: string;
+        start: string;
+        end: string;
+        object_type: string;
+      }>;
+    },
+    callback?: (error: Error | null, data?: { [key: string]: any }) => void
+  ) {
```

Comment:
in case i missed any in the notes above:
```suggestion
  consecutiveAvailability(
    options: {
      emails: Array<string[]>;
      duration: number;
      interval: number;
      start_time?: number;
      startTime?: number;
      end_time?: number;
      endTime?: number;
      free_busy?: Array<{
        email: string;
        object: string;
        time_slots: Array<{
          object: string;
          status: string;
          start_time: number;
          end_time: number;
        }>;
      }>;
      open_hours: Array<{
        emails: Array<string[]>;
        days: number[];
        timezone: string;
        start: number;
        end: number;
        object_type: string;
      }>;
    },
    callback?: (error: Error | null, data?: { [key: string]: any }) => void
  ) {
```

Reply:
Yup, got it in, thank you!